### PR TITLE
IBX-4542: Implemented `ImageAdditionalData` type

### PR DESCRIPTION
**Pull request by barw4**
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | [IBX-4542](https://issues.ibexa.co/browse/IBX-4542)
| **Type**                                   | improvement
| **Target Ibexa version** | `v3.3`
| **BC breaks**                          | no

Example query:
```
{
  media {
    image (id: 261) {
      id,
      name,
      image {
        id,
        fileName,
        fileSize,
        additionalData {
          focalPointX,
          focalPointY
        }
      }
    }
  }
}
```

### CODE REVIEW

**Review by konradoboza — APPROVED** [review]

**Comment by bdunogier** [comment]
What about calling it `focalPoint` ?

File: src/Resources/config/graphql/Field.types.yaml [hunk 1]
```diff
@@ -36,6 +36,9 @@ ImageFieldValue:
             height:
                 type: "Int"
                 resolve: "@=value.height"
+            additionalData:
```
Comment (bdunogier) [inline src/Resources/config/graphql/Field.types.yaml:39]: What about calling it `focalPoint` ?

```
image {
  focalPoint {
    x
    y
  }
}
```

**Review by bdunogier — CHANGES_REQUESTED** [review]

File: src/Resources/config/graphql/Field.types.yaml (hunk 1 above)
Comment (barw4) [reply src/Resources/config/graphql/Field.types.yaml:39]: @bdunogier shouldn't it be on par with what we have in the Image fieldtype? Those values are in fact stored in `additionalData['focalPointX']` and `additionalData['focalPointY']` so this is 1:1 representation.

Comment (adamwojs) [reply src/Resources/config/graphql/Field.types.yaml:39]: Are focalPointX and focalPointY are available in ibexa/oss ?

Comment (barw4) [reply src/Resources/config/graphql/Field.types.yaml:39]: @adamwojs they are not, however, querying them will just return `null`s like shown below:
image
This is the exact same behavior as if we wouldn't have a focal point set on `ibexa/content`.

**Review by adamwojs — APPROVED** [review]

**Review by micszo — APPROVED** [review]
Worked fine for me. 👍🏻 

Tested on Ibexa Commerce 3.3.30-dev.

File: src/Resources/config/graphql/Field.types.yaml (hunk 1 above)
Comment (bdunogier) [reply src/Resources/config/graphql/Field.types.yaml:39]: OK, fine :)

**Pull request merged**